### PR TITLE
Refactor usage sn

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/collections.php [hunk 1]
```diff
@@ -34,28 +34,6 @@
...
-                '$id' => ID::custom('mimeType'),
-                'type' => Database::VAR_STRING,
-                'format' => '',
-                'size' => 255, // https://tools.ietf.org/html/rfc4288#section-4.2
-                'signed' => true,
-                'required' => false,
-                'default' => null,
-                'array' => false,
-                'filters' => [],
-            ],
```
Comment (TorstenDittmann) [inline app/config/collections.php:58]: lets keep that for migration in mind :-)

File: app/init.php [hunk 2]
```diff
@@ -112,7 +112,7 @@
 const APP_KEY_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_USER_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_CACHE_UPDATE = 24 * 60 * 60; // 24 hours
-const APP_CACHE_BUSTER = 330;
+const APP_CACHE_BUSTER = 329;
```
Comment (TorstenDittmann) [inline app/init.php:115]:
```suggestion
const APP_CACHE_BUSTER = 331;
```

File: docker-compose.yml [hunk 3]
```diff
@@ -936,9 +935,8 @@ services:
   #
   # MailCatcher - An SMTP server. Catches all system emails and displays them in a nice UI.
   # RequestCatcher - An HTTP server. Catches all system https calls and displays them using a simple HTTP API. Used to debug & tests webhooks and HTTP tasks
-  # Redis Insight - A nice UI for exploring Redis data
-  # Adminer - A nice UI for exploring MariaDB data
-  # GraphQl Explorer - A nice UI for exploring GraphQL API
+  # RedisCommander - A nice UI for exploring Redis data
+  # Webgrind - A nice UI for exploring and debugging code-level stuff
```
Comment (TorstenDittmann) [inline docker-compose.yml:939]: the comment is wrong now

File: app/config/collections.php (hunk 1 above)
Comment (christyjacob4) [reply app/config/collections.php:58]: There is going to be a follow up PR to add these attributes back. And yes migration is there
https://github.com/appwrite/appwrite/blob/1.5.x/src/Appwrite/Migration/Version/V20.php#L139-L155

File: src/Appwrite/Platform/Tasks/QueueRetry.php [hunk 4]
```diff
@@ -35,21 +37,25 @@ public function __construct()
...
+            ->callback(fn ($name, $limit, $queue) => $this->action($name, $limit, $queue));
     }
 
     /**
      * @param string $name The name of the queue to retry jobs from
+     * @param  mixed $limit
      * @param Connection $queue
      */
-    public function action(string $name, Connection $queue): void
+    public function action(string $name, mixed $limit, Connection $queue): void
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/Appwrite/Platform/Tasks/QueueRetry.php:50]: We need it to be mixed since we use a wildcard validator

File: docker-compose.yml (hunk 3 above)
Comment (christyjacob4) [inline docker-compose.yml:939]:
```suggestion
  # Redis Insight - A nice UI for exploring Redis data
  # Adminer - A nice UI for exploring MariaDB data
  # GraphQl Explorer - A nice UI for exploring GraphQL API
```

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**